cli/cycpp.py

```diff
@@ -89,13 +89,16 @@
               'boost::uuids::uuid', }
 
 BUFFERS = {'{0}::toolkit::ResourceBuff'.format(CYCNS),
+           '{0}::toolkit::ResBuf'.format(CYCNS),
            ('{0}::toolkit::ResBuf'.format(CYCNS), CYCNS + '::Resource'),
            ('{0}::toolkit::ResBuf'.format(CYCNS), CYCNS + '::Product'),
            ('{0}::toolkit::ResBuf'.format(CYCNS), CYCNS + '::Material'),
+           '{0}::toolkit::ResMap'.format(CYCNS),
            }
 
 TEMPLATES = {'std::vector', 'std::set', 'std::list', 'std::pair',
-             'std::map', '{0}::toolkit::ResBuf'.format(CYCNS),}
+             'std::map', '{0}::toolkit::ResBuf'.format(CYCNS),
+             CYCNS + '::toolkit::ResMap',}
```

Comment:
I don't think that we should limit ourselves to just myself, @gidden, and @rwcarlsen as mergers. That is not a healthy community, especially when we have had consistent excessive delays in reviewing, let alone merging. I trust @gonuke, @katyhuff, @rakhimov, @mbmcgarry, @opotowsky, @jdangerx, and all other people with push access just as much as I trust myself.